content/concepts/content-routing/overview.md

```diff
@@ -4,4 +4,56 @@ description: "Learn about Noise in libp2p."
 weight: 218
 ---
 
-Coming soon!
+## Overview
+
+Content routing refers to the process of identifying a specific peer that holds
+certain data and the means to connect to them within a P2P network.
+In a traditional client-server network, routing is a straightforward process
```

Comment:
I think it's more accurate to say something like:

The most common addressing scheme of data today is named addressing. The content you want can be found by some name on a specific server. This makes routing straightforward because you already have the information about which server has the content. e.g. `example.com/cat.jpg` tells us that the server with the domain name `example.com` has a file called `cat.jpg`. This is fine as long as we trust `example.com` to truly give us the correct `cat.jpg`. 

The problem is, what if I want to get some data that when hashed has a specific hash value? This is called content-addressed data because the content of the data itself is the way we address it. The problem now becomes, how do I find out who has this content given that I only have the hash value of the content? This is where content routing comes in.


---

Just a suggestion feel free to reword freely

Reply:
thanks!